### PR TITLE
glog.Infof in stateful

### PR DESCRIPTION
**Pull request by jianglingxia**
**What this PR does / why we need it**:
in stateful there has two place that need not Infof and fix it
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Review by krmayankk — APPROVED** [review]

**Comment by jianglingxia** [comment]
can you label /approve no-issue？thanks very much @krmayankk

**Comment by jianglingxia** [comment]
/assign @enisoc
PTAL thanks

**Comment by jianglingxia** [comment]
PTAL @foxish @enisoc

File: pkg/controller/statefulset/stateful_set.go [hunk 1]
```diff
@@ -143,8 +143,8 @@ func (ssc *StatefulSetController) Run(workers int, stopCh <-chan struct{}) {
 	defer utilruntime.HandleCrash()
 	defer ssc.queue.ShutDown()
 
-	glog.Infof("Starting stateful set controller")
-	defer glog.Infof("Shutting down statefulset controller")
+	glog.Info("Starting stateful set controller")
```
Comment (enisoc) [inline pkg/controller/statefulset/stateful_set.go]: While you're at it, could you fix the text to be consistent for these two lines? Let's call it "StatefulSet controller".

Comment (jianglingxia) [reply pkg/controller/statefulset/stateful_set.go]: ok,thanks,I have fixed it,for your review!@enisoc

**Comment by jianglingxia** [comment]
I have signed CLA @k8s-ci-robot

**Comment by jianglingxia** [comment]
@enisoc  PTAL for your approve thanks

**Comment by jianglingxia** [comment]
reopen 53877, because of some reason the branch become unknow ,that I cannot squash the commits into one! so I close the pr,thanks!

**Pull request closed without merging**